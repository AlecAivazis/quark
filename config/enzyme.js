// external imports
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import browserEnv from 'browser-env'

// add a request animation frame polyfill
import raf from 'raf'
raf.polyfill()

// enable the react 16 adapter for enzyme
Enzyme.configure({ adapter: new Adapter() })

// add various browser mocks
browserEnv()
