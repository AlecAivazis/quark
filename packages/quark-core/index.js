export * from './components';
import * as _styles from './styles';
export { _styles as styles };