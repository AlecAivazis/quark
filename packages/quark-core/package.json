{
    "name": "quark-core",
    "version": "1.0.6",
    "module": "index.js",
    "main": "index.js",
    "dependencies": {
        "react-native-web": "0.3.4",
        "svgs": "^3.1.1"
    },
    "peerDependencies": {
        "react": "^16.3.0"
    }
}
