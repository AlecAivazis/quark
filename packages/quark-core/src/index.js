// @flow
export * from './components'
export * as styles from './styles'
