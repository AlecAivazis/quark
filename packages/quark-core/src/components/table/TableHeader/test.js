// external imports
import React from 'react'
import { Text } from 'react-native'
import { mount } from 'enzyme'
// local imports
import { TableHeader } from 'quark-core'

it('renders', () =>
    expect(
        mount(
            <TableHeader>
                <Text>hello</Text>
            </TableHeader>
        )
    ).toMatchSnapshot())
