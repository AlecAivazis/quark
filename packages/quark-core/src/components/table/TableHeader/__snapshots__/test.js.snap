// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<TableHeader>
  <GetTheme>
    <TableRow
      style={
        Object {
          "borderColor": "#C2C2C2",
          "cursor": "pointer",
          "touchAction": "manipulate",
        }
      }
    >
      <GetTheme>
        <View
          style={
            Object {
              "backgroundColor": "transparent",
              "borderColor": "#C2C2C2",
              "cursor": "pointer",
              "touchAction": "manipulate",
            }
          }
        >
          <div
            className="rn-alignItems-1oszu61 rn-borderBottomWidth-qklmqi rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flexShrink-1pxmb3b rn-flexBasis-7vfszb rn-flexDirection-18u37iz rn-height-eu3ka rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingRight-9aemit rn-paddingLeft-gy4na3 rn-paddingBottom-14p8ni rn-paddingTop-zre7wq rn-position-bnwqim rn-zIndex-1lgpqti"
            style={
              Object {
                "borderBottomColor": "rgba(194,194,194,1)",
                "borderLeftColor": "rgba(194,194,194,1)",
                "borderRightColor": "rgba(194,194,194,1)",
                "borderTopColor": "rgba(194,194,194,1)",
              }
            }
          >
            <Text>
              <div
                className="rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-color-homxoj rn-display-1471scf rn-font-1lw9tu2 rn-fontFamily-10u92zi rn-fontSize-1b43r93 rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-textDecoration-bauka4 rn-whiteSpace-q42fyq rn-wordWrap-qvutc0"
                dir="auto"
              >
                hello
              </div>
            </Text>
          </div>
        </View>
      </GetTheme>
    </TableRow>
  </GetTheme>
</TableHeader>
`;
